src/ceph-volume/ceph_volume/devices/lvm/common.py

```diff
@@ -56,6 +56,19 @@ def common_parser(prog, description):
         help='OSD data path. A physical device or logical volume',
     )
 
+    required_group.add_argument(
+        '--data-size',
+        help='Size of data LV in case a device was passed in --data',
```

Comment:
Hm. I assume this is for underprovisioning devices? What happens if I specify devices with different sizes in `--data`?

Reply:
This is for create/prepare. These commands only create single OSDs.